.github/workflows/pr_test.yml

```
@@ -0,0 +1,76 @@
+on: pull_request
+name: Test
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v1
```

See setup-go comments for reasoning

```suggestion
      uses: actions/checkout@v2
```